build/constants.yaml

```diff
@@ -215,8 +215,8 @@
     gem-version: 3.3.10
     ruby31-version: 3.1.1
     ruby31-tar-sha256: 7aefaa6b78b076515d272ec59c4616707a54fc9f2391239737d5f10af7a16caa
-    ruby27-version: 2.7.2
-    ruby27-tar-sha256: 6e5706d0d4ee4e1e2f883db9d768586b4d06567debea353c796ec45e8321c3d4
+    ruby27-version: 2.7.6
+    ruby27-tar-sha256: 54dcd3044726c4ab75a9d4604720501442b229a3aed6a55fe909567da8807f24
```

Comment:
I think the ruby tar.gz sha256 should be `e7203b0cc09442ed2c08936d483f8ac140ec1c72e37bb5c401646b7866cb5d10` based on https://www.ruby-lang.org/en/news/2022/04/12/ruby-2-7-6-released/